templates/flavors/external-cloud-provider-windows/cloud-controller-manager.yaml

```diff
@@ -0,0 +1,198 @@
+apiVersion: v1
```

Comment:
should these files come from the cloud manage repository so any changes made to them would be reflected in the tests?

Reply:
we have some slight customizations in the cloud-controller-manager container, eg. image and clusterName. Do you know if kustomize let's you take a file url as base?

Side note, this should not have been duplicated in this PR though, I copied it into the new directory by accident, it should use the same file as we were using before.

Reply:
This is an example of using remote resources: https://github.com/kubernetes-sigs/kustomize/blob/master/examples/loadHttp.md. Admittedly haven't tried it my self

Reply:
have someone confirmed whether the remote resources work or not?

Reply:
@jsturtevant I'm using the files in a ConfigMapGenerator, not as a resource, so I'm not sure I can use the remove resource there